site/content/english/blog/BlockSplit3-recap.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "BlockSplit 3"
+date: 2022-06-07
+draft: false
+image: "https://miro.medium.com/max/1400/0*yXZPgVCpuKh2WiXc"
```

Comment:
reupload this picture and host it on our website, we don't want dead links in the future :)

Reply:
also, seems very low quality